### DISCUSSION TITLE
Τελική παρουσίαση

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/hci#1648] [Announcements]
Θα ήθελα να ρωτήσω, εάν όπως αναφέρεται, η 2η παρουσίαση προόδου θα γίνει την 10η εβδομάδα (13-19 Δεκεμβρίου), πιο πρίν ή πιο μετά? Ρωτάω, διότι λόγω οτι πλησιάζουν οι διακοπές, πιστεύω πως αρκετά άτομα έχουμε αυτήν την απορία, οπότε θα ήταν μια διευκόλυνση το να γνωρίζουμε πότε θα γίνει, ώστε να κανονίσουμε να μη λείπουμε.

**Comment by p20sami** [comment]
@epidrome Κάποια απάντηση? Υπάρχουν υποχρεώσεις κοντά στη περίοδο αυτήν και πρέπει να ξέρω (και άλλοι φαντάζομαι) ώστε να διευθετηθούν για να ξέρω να μην λείπω στη παρουσίαση.

**Reply by p14kairas** [reply]
Ναι και εγώ θα ήθελα να το γνώριζα όσο το δυνατόν πιο σύντομα.

**Comment by epidrome** [comment]
γενικά, αν έχεις παρατηρήσει, αυτόν τον καιρό, δύσκολο να προβλέψουμε τι θα γίνει σε βάθος χρόνου πάνω από μια εβδομάδα

όπως πάντα, θα βγει έγκαιρα σχετική ανακοίνωση, 

πάντως με δεδομένη την πολύ μικρή (και κάπως χλιαρή #1644) συμμετοχή στην ενδιάμεση παρουσίαση, η τελική μάλον θα πάει για μετά τις γιορτές στις εβδομάδες 11-12 και την εβδομάδα 10 θα γίνουν μόνο συναντήσεις-φροντιστήρια
